docs/core/deploying/index.md

```diff
@@ -1,95 +1,120 @@
+01. A platform-specific executable.
 
-- Self-contained deployment. Unlike FDD, a self-contained deployment (SCD) doesn't rely on the presence of shared components on the target system. All components, including both the .NET Core libraries and the .NET Core runtime, are included with the application and are isolated from other .NET Core applications. SCDs include an executable (such as *app.exe* on Windows platforms for an application named `app`), which is a renamed version of the platform-specific .NET Core host, and a *.dll* file (such as *app.dll*), which is the actual application.
+    Depending on the commands used, platform-specific executables can be published [self-contained](#self-contained) or [runtime-dependent](#runtime-dependent).
 
-- Framework-dependent executables. Produces an executable that runs on a target platform. Similar to FDDs, framework-dependent executables (FDE) are platform-specific and aren't self-contained. These deployments still rely on the presence of a shared system-wide version of .NET Core to run. Unlike an SCD, your app only contains your code and any third-party dependencies that are outside of the .NET Core libraries. FDEs produce an executable that runs on the target platform.
+    Commands that produce an executable:
 
-## Framework-dependent deployments (FDD)
+    | Type                                    | SDK Version   | Command |
+    | --------------------------------------- | ------------- | ------- |
+    | [runtime-dependent](#runtime-dependent) | 3.1, 3.0      | [`dotnet publish`](../tools/dotnet-publish.md) |
+    |                                         | 2.1           | Not supported. |
+    | [self-contained](#self-contained)       | 3.1, 3.0, 2.1 | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
 
-For an FDD, you deploy only your app and third-party dependencies. Your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core and ASP.NET Core apps that target .NET Core.
+    In general, platform-specific apps can be created for platforms other than your current platform. The `-r <RID>` switch sets the target [runtime-identifier](../rid-catalog.md).
 
-### Why create a framework-dependent deployment?
+01. A runtime-dependent cross-platform *dll* file.
 
-Deploying an FDD has a number of advantages:
+    Commands that produce a runtime-dependent *dll* app:
```

Comment:
I think it's better to call this an "app" as it's not a library which you would assume with the .dll extension. It has an entry point and is meant to be run. I'm not totally sure how to articulate this though, it's a confusing file extension to be chosen...

Thoughts?

Reply:
The wording has .dll as an adjective modifying app, as if .dll is a kind of app. Maybe "... produce a runtime-dependent app as a .dll" would be clearer?

Reply:
Hrm. Maybe it's better to not call out the extension unless talking about the file directly. Instead, could use the terms "platform-specific executable" and "cross-platform binary" to truly differentiate?

Reply:
Maybe it isn't necessary to call out the file type at this level.  Maybe your main point is the difference between platform-specific (which is itself a misnomer since it's really runtime-specific) and cross-platform, and you could deal with file types inside the respective sections.  However, you would still have the confusing mismatch between intro and body, where intro is emphasizing cross-platform vs. platform-specific, while the body is emphasizing the difference between self-contained and runtime-dependent.

Reply:
Originally this article was a bit more feature driven (create a FDD, FDE, SCD (framework-dep-deployment, frame-dep-exe, self-contained-exe)) but people are more thinking more about `How do I want a user to start my app: dotnet file.dll or file.exe?` However, if I align with that then there is a lot more repeating info as runtime-dependent spans both dll and exe formats. I think the way it is now lessens how much info is repeated. However, maybe I can redo the table pivot or do a more proper intro after the table pivot.